addons/mail/static/src/models/messaging_notification_handler/messaging_notification_handler.js

```diff
@@ -107,6 +107,8 @@ function factory(dependencies) {
                             return this._handleNotificationRtcSessionEnded(message.payload);
                         case 'res_users_settings_volumes_update':
                             return this._handleNotificationVolumeSettingUpdate(message.payload);
+                        case 'attachment_deleted':
```

Comment:
name it `mail.attachment_delete` (similar name as the js model, and with the name of the method to call after `_`)